.github/wordlist.txt

```diff
@@ -41,36 +44,44 @@ csv
 customizable
 dataset
 datasets
+devnull
 dicts
 df
 dtype
 dunder
 env
 geopoint
 geospatial
+glpk
 gpd
 inplace
 ipynb
 ipywidgets
 iterable
 json
 libs
+msg
 multicommodity
 ndarray
 np
 num
```

Comment:
Could you please add `numpy` to this list?

Reply:
will do!